packages/millicast-sdk/src/Publish.js

```diff
@@ -169,6 +169,11 @@ export default class Publish extends BaseWebRTC {
     }
   }
 
+  stop () {
+    super.stop()
+    this.worker?.terminate()
```

Comment:
Do we need to do this?
```suggestion
    this.worker?.terminate()
    this.worker = null
```